### DISCUSSION TITLE
Which file do I need to start?

### DISCUSSION THREAD

**Discussion by FinlayHDev** [requarks/wiki#4034] [Help / Questions] [answered]
What file do I need to start?
<img width="1479" alt="Screenshot 2021-05-29 at 15 08 17" src="https://user-images.githubusercontent.com/73852683/120073454-49264300-c090-11eb-9482-e01be40d6eb3.png">

This is what I'm trying to achieve
<img width="1415" alt="Screenshot 2021-05-29 at 15 13 23" src="https://user-images.githubusercontent.com/73852683/120073527-999da080-c090-11eb-8a19-ff48287e8c08.png">

...but I need to know the file to start with for it to work
Thanks for any help in advance

**Comment by NGPixel** [comment, marked as answer]
The startup file is `server/index.js`
